RELEASE_NOTES.md

```diff
@@ -29,8 +29,9 @@ Refer to the [upgrading guide](https://github.com/cosmos/interchain-security/blo
 <!-- Add any highlights of this release -->
 
 ## ❤️ Contributors
-
+<!-- markdown-link-check-disable -->
 * Informal Systems ([@informalinc](https://twitter.com/informalinc))
```

Comment:
Tiny change because PR was failing on markdown-link checking.
We could also use [Nitter](https://nitter.net/informalinc) instead but it's quite likely for Nitter to be down once in a while (thanks Elon!).